clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -485,6 +485,12 @@ static void InitializeStandardPredefinedMacros(const TargetInfo &TI,
 
     if (LangOpts.SYCLValueFitInMaxInt)
       Builder.defineMacro("__SYCL_ID_QUERIES_FIT_IN_INT__", "1");
+
+    // Enable __SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ macro for
+    // all FPGA compilations.
```

Comment:
This statement of `all FPGA compilations` is not true if you are only checking against `-fintelfpga`.  Using the triple is valid for FPGA as well (using `-fiintelfpga` will also imply the spir64_fpga triple)

Reply:
Thanks @mdtoguchi.  I have updated comments since I am  setting __SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ macro for  both host and device compilations if -fintelfpga flag is present.